addons/mail/models/mail_channel.py

```diff
@@ -180,46 +211,54 @@ def _onchange_moderation(self):
 
     @api.model_create_multi
     def create(self, vals_list):
-        defaults = self.default_get(['image_128'])
-        current_partner = self.env.user.partner_id.id
+        defaults = self.default_get(['image_128', 'public'])
 
-        visibilities = []
+        access_types = []
         for vals in vals_list:
+            # find partners to add from partner_ids
```

Comment:
Manual ORM commands manipulation, wow :(

Reply:
That's what Odoo developers from 2011 do.